sections/Navbar.jsx

```diff
@@ -12,10 +12,19 @@ import { baseTheme } from "../theme";
 import { media } from "../utils";
 
 {
-  /* TODO: Refactor into object? paths must be same as linked pages*/
+  /* @TODO: Refactor into object? paths must be same as linked pages */
+  // done i think?
 }
+
+// map array to preserve order
 const RESOURCES = ["Home", "About Us", "Resources", "Events", "Community"];
-const PATHS = ["", "about", "resources", "events", "community"];
+const PATH_NAMES = {
+  Home: "",
+  "About Us": "about",
+  Resources: "resources",
+  Events: "events",
+  Community: "community",
```

Comment:
nit: almost prefer for consistency that the keys all be strings - would this affect anything?

Reply:
do you think there is a way to use RESOURCES as the key names here? seems redundant to have both